classes/simplemap.php

```diff
@@ -787,7 +787,7 @@ function load_simplemap( lat, lng, aspid, ascid, asma, shortcode_zoom_level, map
 			<?php
 			if ( '' == $options['api_key'] ) {
 				?>
-				jQuery( "#simplemap" ).html( "<p style='padding:10px;'><?php printf( __( 'You must enter an API Key in <a href=\"%s\">General Settings</a> before your maps will work.', 'simplemap' ), admin_url( 'admin.php?page=simplemap' ) ); ?></p>" );
+				jQuery( "#simplemap" ).html( "<p style='padding:10px;'><?php printf( __( 'You must enter an API Key in <a href=\"%s\">General Settings</a> before your maps will work.', 'SimpleMap' ), admin_url( 'admin.php?page=simplemap' ) ); ?></p>" );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 253 characters

<!-- Comment by [SideCI](https://sideci.com) -->